backend/services/notification_service.py

```
@@ -0,0 +1,13 @@
+from sqlalchemy.orm import Session
+from models.user import User
+from models.post import Post
+from core.connection_crud import get_connections
+from crud.notification import create_notification
+
+def send_post_notifications(db: Session, user: User, post: Post):
```

Ruff (0.8.2)

Missing return type annotation for public function `send_post_notifications`

Add return type annotation: `None`

(ANN201)